bundle-workflow/src/run_integ_test.py

```diff
@@ -29,13 +29,19 @@
 def parse_arguments():
     parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
     parser.add_argument(
-        "--bundle-manifest", type=argparse.FileType("r"), help="Bundle Manifest file."
+        "--s3-bucket", type=str, help="S3 bucket name"
     )
     parser.add_argument(
-        "--build-manifest", type=argparse.FileType("r"), help="Build Manifest file."
+        "--opensearch-version", type=str, help="OpenSearch version to test"
```

Comment:
It would be nice to have the suite configurable to just run integration tests in a single component. May be take `--component` as a parameter.

Reply:
+1 to that. Will take this up as an enhancement separately